### PR TITLE
chore: make vue-bootstrap an external dependency

### PR DESCRIPTION
**Pull request by jvanveen**
* fix components-library tests
* tweaked build performance
* cleanup third-party dependencies ( remove from deps, client lib should provide these)
* update readme

#### Checklist
- [ ] Functionality works & meets specifications
- [ ] Code reviewed
- [ ] Code unit/integration/system tested
- [ ] User documentation updated
- [ ] Conventional commits (squash if needed)
- [ ] No warnings during install
- [ ] Updated javascript typing

### CODE REVIEW

**Comment by jvanveen** [comment]
[#7576 Gebruik components-library in DE2](https://trello.com/c/YtJWuRMu/308-7576-gebruik-components-library-in-de2)

File: packages/data-explorer/src/main.ts [hunk 1]
```diff
@@ -1,12 +1,27 @@
...
 import App from './App.vue'
 import router from './router'
 import store from './store/store'
 
 // @ts-ignore
 import i18n from '@molgenis/molgenis-i18n-js'
-import 'bootstrap-vue/dist/bootstrap-vue.css'
+
+import '@molgenis-ui/components-library/src/icons'
+import devDependencies from '@molgenis-ui/components-library/src/dev-dependencies'
```
Comment (jvanveen) [inline packages/data-explorer/src/main.ts:11]: Doesn't belong here?

File: packages/components-library/src/components/ui/CartSelectionToast.vue [hunk 2]
```diff
@@ -45,7 +45,6 @@
 
 <script>
 export default {
-  name: 'CartSelectionToast',
```
Comment (connoratrug) [inline packages/components-library/src/components/ui/CartSelectionToast.vue:48]: suggest to put back the name

File: packages/components-library/src/components/filters/ChangeFilters.vue [hunk 3]
```diff
@@ -149,23 +141,27 @@ export default Vue.extend({
...
-  width: 1.5em;
-  text-align: center;
-  display: inline-block;
-  position: absolute;
-  right: 10px;
-  cursor: pointer;
-}
-.remove-button:hover {
-  color: var(--danger);
+<style lang="scss">
```
Comment (connoratrug) [inline packages/components-library/src/components/filters/ChangeFilters.vue:147]: why remove the scoped ? is this needed to use rollup and scss ?

Comment (jvanveen) [reply packages/components-library/src/components/filters/ChangeFilters.vue:147]: Reverting

**Review by connoratrug — APPROVED** [review]

**Pull request merged**